### ISSUE TITLE
Support for pyproject.toml

### ISSUE THREAD

**Issue by joshbode** [koxudaxi/datamodel-code-generator#188]
`datamodel-codegen` is gaining a lot of command-line options which is great, but it can make the invocation quite lengthy.

How about adding support for a `pyproject.toml` section, e.g.
```toml
[tool.datamodel-codegen]
field-constraints = true
snake-case-field = true
strip-default-none = false
target-python-version = "3.7"
```

I'm happy to add this if there is desire for this feature :)

**Comment by koxudaxi** [comment]
@joshbode 
I agree.
I recently want the same feature :smile: 

I will implement it :rocket:

**Comment by koxudaxi** [comment]
@joshbode 
I have implemented the feature and released a new version as  `0.5.23`.

**Comment by joshbode** [comment]
Fantastic - thank you!
I'll try it out today :)
